### DISCUSSION TITLE
Using react-select inside a useFieldArray

### DISCUSSION THREAD

**Discussion by blaropal** [react-hook-form/react-hook-form#6866] [Q&A] [answered]
I am using react-select inside a useFieldArray, the problem I have is when you have multiple rows and one gets removed the value of the select is not updated to the value of the next row as expected. 
It holds the value of the row that was just removed instead.
[CodeSandbox](https://codesandbox.io/s/great-burnell-dl4yq)

Any ideas on how to solve this ?

**Comment by bluebill1049** [comment, marked as answer]
are you missing this?

`<React.Fragment key={item.id}>`

**Reply by blaropal** [reply]
Had to be something as simple as that, thank you!
